src/data-display/dynamic/dynamic-layout/templates/raw-table/raw-table.stories.mdx

```diff
@@ -107,6 +108,79 @@ export const Template = (args, { argTypes }) => ({
 | click-settings | X |
 | update-popup-visible | X |
 
+<br/>
+<br/>
+
+# Options Example
+
+<br/>
```

Comment:
suggestion: I think 'Options Example' must be under the 'Raw Table Type'.
What about use h2 for 'Options Example'  and use h3 for '## Options: Disable Search' and '## Options: Headers'?